docs/sources/slack/example-api-responses/sanitized/discovery-users-list.json

```diff
@@ -13,23 +12,14 @@
         "team":"T02DDBANBM5"
       },
       "is_admin":true,
-      "is_owner":true,
-      "is_primary_owner":false,
-      "is_restricted":false,
-      "is_ultra_restricted":false,
-      "is_bot":false,
```

Comment:
should we let `is_bot` pass always?

Reply:
yes; it was passing as `bot`; there's a `@JsonAlias("is_bot")` on the property, but seems to not be coming through;